interceptor/interceptor.go

```diff
@@ -222,6 +222,47 @@ type ClientUpdateWorkflowInput = internal.ClientUpdateWorkflowInput
 // ClientOutboundInterceptor.UpdateWithStartWorkflow.
 type ClientUpdateWithStartWorkflowInput = internal.ClientUpdateWithStartWorkflowInput
 
+// NexusOperationInboundInterceptor is an interface for intercepting Nexus operation method invocations.
+//
+// All implementations must embed [NexusOperationInboundInterceptorBase] to safely handle future changes.
+//
+// Note: Experimental
```

Comment:
I marked interceptors as experimental at this stage. Ideally we would have at least one release to let this bake. I don't think this is a blocker for GA.